Source/WebCore/page/Quirks.cpp

```diff
@@ -721,6 +721,21 @@ bool Quirks::needsFullscreenDisplayNoneQuirk() const
 #endif
 }
 
+bool Quirks::needsFullscreenObjectFitQuirk() const
```

Comment:
Can you add this comment:
`// cnn.com rdar://119640248`

Makes @karlcow's life easier so he can more easily audit what quirks can be removed.

Reply:
me or anyone in QA who's less savy with Blame and multiple rewrite history.